### PR TITLE
[WIP] Create node server

### PR DESCRIPTION
**Pull request by sergiga**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const app = express();
+
+app.get('/', (req, res) => {
+    res.status(200).send('Hello World!')
```
Comment (fosiperalta) [inline app.js]: Status should be 201

Comment (sergiga) [inline app.js]: Semicolon missing

File: app.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const app = express();
```
Comment (fosiperalta) [inline app.js]: missing `;`

**Review by sergiga — COMMENTED** [review]
Refactoring bugs

**Review by fosiperalta — CHANGES_REQUESTED** [review]

**Review by fosiperalta — APPROVED** [review]

**Pull request closed without merging**